ui/src/app/cohort-search/cohort-search/cohort-search.component.tsx

```diff
@@ -377,15 +377,15 @@ export const CohortSearch = fp.flow(
         parentId: null,
         parameterId: '',
         type: CriteriaType.PPI.toString(),
-        name: 'Whole Genome Sequence',
+        name: 'Short Read Whole Genome Sequence',
```

Comment:
Do we want to name this to match the menu item names: 'Short Read WGS'? It will fit in the group box better as well. This could also apply to 'Long Read WGS' as well

Reply:
Works for me. Yeah, I assume researchers will know what WGS means, no need to spell it out.